osu.Framework/Platform/Clipboard.cs

```diff
@@ -20,7 +20,15 @@ public abstract class Clipboard
         /// Copy text to the clipboard.
         /// </summary>
         /// <param name="text">Text to copy to the clipboard</param>
-        public abstract void SetText(string text);
+        public void SetText(string text)
+        {
+            SetData(
+                new ClipboardData
+                {
+                    Text = text
+                }
+            );
```

Comment:
What's with the spacing?

```suggestion
            SetData(new ClipboardData { Text = text });
```